src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/advice/InvalidRequestException.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.reform.cwrdapi.controllers.advice;
+
+public class InvalidRequestException extends RuntimeException {
+
+    public InvalidRequestException(String message) {
+        super(message);
```

Comment:
can we please rename this as CaseworkerApiException and use everywhere to throw this.

Reply:
this is specific to invalidRequest so i can't use same in every where..status code varies in each scenario